integ/scripts/bash/rfdk-integ-e2e.sh

```diff
@@ -71,6 +71,12 @@ export INFRASTRUCTURE_DEPLOY_FINISH_TIME=$SECONDS
 
 # Pull the top level directory for each cdk app in the components directory
 for COMPONENT in **/cdk.json; do
+    # Invoke hook function if it is exported and name is defined in HOOK_FUNCTION variable
+    if [ -n "$HOOK_FUNCTION" ]  && [ "$(type -t $HOOK_FUNCTION)" == "function" ]
+    then
+      $HOOK_FUNCTION
```

Comment:
Is there any argument/parameters that might be useful to pass to the hook function? I'm trying to think in the general case if someone would need a hook function that is triggered before each component test, they'd likely be interested in knowing which component is being tested. Does it make sense to include the component name?